libraries/tools/kotlin-gradle-plugin/src/functionalTest/kotlin/org/jetbrains/kotlin/gradle/SyncKotlinAndAndroidSourceSetsTest.kt

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+@file:Suppress("invisible_reference", "invisible_member", "FunctionName", "DuplicatedCode")
```

Comment:
(minor) Is this used for accessing `internal`s from the `main` source set while we don't have working associate compilations in the IDE? If so, then it's OK, but overall, suppressing this looks quite scary. Could you please add a comment explaining the purpose of this suppression?

Reply:
Yes, non-supported associate compilations is the reason 😞 
Added a comment to all occasions of this suppress ✅